internal/kldeth/send.go

```diff
@@ -155,13 +161,14 @@ type sendTxArgs struct {
 }
 
 // sendUnsignedTxn sends a transaction for internal signing by the node
-func (tx *Txn) sendUnsignedTxn(ctx context.Context, rpc RPCClient, txArgs *sendTxArgs) (string, error) {
+func (tx *Txn) sendUnsignedTxn(ctx context.Context, rpc RPCClient, txArgs *SendTXArgs) (string, error) {
```

Comment:
should this function now be renamed, since we `Sign()` the tx (if a `Signer` is present)?